autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -35,39 +36,58 @@ const ConfigureCard: React.FC = () => {
     const courses = [];
     for (let i = 0; i < courseCards.numCardsCreated; i++) {
       if (courseCards[i] && courseCards[i].course) {
-        const [subject, courseNum] = courseCards[i].course.split(' ');
+        const courseCard = courseCards[i];
+
+        const selectedSections: string[] = []; // the section nums of the selected sections
+
+        // Iterate through the sections and only choose the ones that are selected
+        courseCard.sections.forEach((sectionSel) => {
+          if (sectionSel.selected) {
+            selectedSections.push(sectionSel.section.sectionNum);
+          }
+        });
+
+        const [subject, courseNum] = courseCard.course.split(' ');
+        const isBasic = courseCard.customizationLevel === CustomizationLevel.BASIC;
+
+        // The default option for honors and web when the Section customization level is selected
+        const filterDefault = 'no_preference';
+
         courses.push({
           subject,
           courseNum,
-          sections: courseCards[i].sections,
-          honors: courseCards[i].honors,
-          web: courseCards[i].web,
+          sections: isBasic ? [] : selectedSections, // Only send if "Section" customization level
+          // Only send if "Basic" level
+          honors: isBasic ? courseCard.honors ?? filterDefault : filterDefault,
```

Comment:
Can I get some parentheses around this to make it easier to read? Side note, nice use of the new `??` operator